### DISCUSSION TITLE
Might the Cluster Discovery feature use on production environment？

### DISCUSSION THREAD

**Discussion by jincong8973** [ClickHouse/ClickHouse#68444] [Q&A]
cluster discovery is a attractive way to config a cluster, but it's maked as experimental feature, is that practiced on production environment?
thanks
[https://clickhouse.com/docs/en/operations/cluster-discovery](url)

**Comment by qoega** [comment]
It is not used ClickHouse Cloud, but it may work correctly and be production ready. We did not get enough feedback from usage in production to switch it to non-experimental.
